gdpr/vendorlist-fetching.go

```diff
@@ -27,8 +27,27 @@ type saveVendors func(uint16, api.VendorList)
 // Nothing in this file is exported. Public APIs can be found in gdpr.go
 
 func newVendorListFetcher(initCtx context.Context, cfg config.GDPR, client *http.Client, urlMaker func(uint16, uint8) string, TCFVer uint8) func(ctx context.Context, id uint16) (vendorlist.VendorList, error) {
+	var defaultVL api.VendorList = nil
+
+	if TCFVer == tCF1 && len(cfg.TCF1.DefaultGVLPath) > 0 {
+		defaultVLbody, err := ioutil.ReadFile(cfg.TCF1.DefaultGVLPath)
+		if err != nil {
+			glog.Fatalf("Error reading from file %s: %v", cfg.TCF1.DefaultGVLPath, err)
```

Comment:
Should this method return an error as well if we can't load the default GVL? Same with the result of the `ParseEagerly` call.